vosekast_control/Scale.py

```diff
@@ -44,6 +44,7 @@ def __init__(
         self.vosekast = vosekast
         self.state = States.NONE
         self.mqtt = self.vosekast.mqtt_client
+        self.scale_publish = False
```

Comment:
Whats the meaning of that new variable?

Reply:
scale_publish allows the scale to only publish values if the input is valuable.